docs/v3/Destination.md

```
@@ -0,0 +1,24 @@
+| Name                | Type     | Description                                                                                                                              |
+| ------------------- | -------- | ---------------------------------------------------------------------------------------------------------------------------------------- |
+| `destination`       | `string` | Destination for the campaign. See [Destination](/src/enums/Destination.ts)                                                               |
+| `threshold`         | `number` | The threshold for the destination, which determines the minimum number of orders required to trigger the delivery. Minimum value is `1`. |
+| `product`           | `string` | The product used for this destination. See [Format](/src/enums/Format.ts)                                                                |
+| `finish`            | `string` | The finish type for the product used in this destination. See [Finish](/src/enums/Finish.ts)                                             |
+| `designId`          | `string` | (Optional) The ID of an existing design used for this destination.                                                                       |
+| `variablesFallback` | `object` | (Optional) Personalization data fallback values. Can contain string, number, or boolean values.                                          |
+
+## One-off
+
+In addition to the [Continuous](#Continuous) fields, One-off has these fields too.
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid
Expected: `#continuous`; Actual: `#Continuous`

(MD051, link-fragments)